### DISCUSSION TITLE
Drag and drop interaction with desktop possible?

### DISCUSSION THREAD

**Discussion by mmMike** [emilk/egui#377] [Q&A]
Hi folks,

is it possible to drag custom mime date to the outside of an egui app (desktop or web)? 
And the other way round, is it possible to receive drops from the outside of an egui app (desktop or web)?

**Comment by emilk** [comment]
Dropping files into `egui_glium` should be fairly easy to implement (relevant issue: https://github.com/emilk/egui/issues/270). Dragging things _out of an egui app_, I don't know. That seems harder

**Reply by mmMike** [reply]
Thank you, besides files I would need text and hyperlinks.
